scss/_layouts_fluid-breakout.scss

```
@@ -0,0 +1,207 @@
+  }
+
+  %l-fluid-breakout__aside--right-ie-11-fallback#{$suffix} {
+    @extend %l-fluid-breakout__aside-common-properties-ie-11-fallback#{$suffix};
+
+    @media (max-width: $l-fluid-breakout-3-col-threshold) {
+      padding-left: 0;
+    }
+  }
+
+  .l-fluid-breakout#{$suffix} {
+    @extend %l-fluid-breakout-ie11-fallback#{$suffix};
+
+    @supports (display: grid) {
+      display: block;
+      grid-gap: 0 0;
+    }
+
+    @media (min-width: $breakpoint-large) {
+      margin-left: auto;
+      margin-right: auto;
+      max-width: 2 * $l-fluid-breakout-aside-width + $l-fluid-breakout-max-width;
+
+      @supports (display: grid) {
+        display: grid;
+        grid-template-columns:
+          minmax($l-fluid-breakout-aside-width, 1fr)
+          minmax(0, $l-fluid-breakout-max-width)
+          minmax($l-fluid-breakout-aside-width, 1fr);
+        grid-template-rows: auto;
+      }
+    }
+
+    @media (max-width: $threshold-4-6-col) {
+      padding-left: map-get($grid-margin-widths, small);
+      padding-right: map-get($grid-margin-widths, small);
+    }
+
+    @media (min-width: $threshold-4-6-col) and (max-width: $threshold-6-12-col) {
+      padding-left: map-get($grid-margin-widths, medium);
+      padding-right: map-get($grid-margin-widths, medium);
+    }
+
+    @media (min-width: $threshold-6-12-col) and (max-width: $l-fluid-breakout-3-col-threshold) {
+      padding-left: map-get($grid-margin-widths, large);
+      padding-right: map-get($grid-margin-widths, large);
+    }
+
+    @media (min-width: $l-fluid-breakout-3-col-threshold) {
+      padding-left: 0;
+      padding-right: 0;
+
+      @supports (display: grid) {
+        grid-gap: 0 0;
+      }
+    }
+
+    .l-fluid-breakout__main {
+      @extend %l-fluid-breakout__main-ie11-fallback#{$suffix};
+
+      @supports (display: grid) {
+        display: grid;
+        grid-gap: 0 $grid-gap;
+        grid-row: 1;
+        grid-template-columns: repeat(auto-fit, minmax($l-fluid-breakout-main-child-width, 1fr));
+        width: 100%;
+      }
+
+      @media (min-width: $l-fluid-breakout-3-col-threshold) {
+        padding-left: map-get($grid-margin-widths, large);
+        padding-right: map-get($grid-margin-widths, large);
+      }
+
+      @media (min-width: $breakpoint-large) {
+        @supports (display: grid) {
+          grid-column: auto / span 2;
+
+          // stylelint-disable-next-line max-nesting-depth
+          &.no-aside {
+            grid-column: 2 / span 2;
+          }
+        }
+      }
+
+      &.is-full-width {
+        @media (min-width: $breakpoint-large) {
+          // stylelint-disable-next-line max-nesting-depth
+          @supports (display: grid) {
+            grid-column: 1 / span 3;
+          }
+        }
+      }
+
+      .row {
+        max-width: 100%;
+        padding-left: 0;
+        padding-right: 0;
+      }
+    }
+
+    .l-fluid-breakout__main-item {
```

For simplicity, can we rename it to `l-fluid-breakout__item`? I don't think we need to nest under main .

```suggestion
    .l-fluid-breakout__item {
```